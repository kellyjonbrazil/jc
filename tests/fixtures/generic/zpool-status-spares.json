[{"pool":"tank","state":"ONLINE","scan":"scrub repaired 0B in <snip>","config":[{"name":"tank","state":"ONLINE","read":0,"write":0,"checksum":0},{"name":"mirror-0","state":"ONLINE","read":0,"write":0,"checksum":0},{"name":"sdh","state":"ONLINE","read":0,"write":0,"checksum":0},{"name":"sdk","state":"ONLINE","read":0,"write":0,"checksum":0},{"name":"mirror-1","state":"ONLINE","read":0,"write":0,"checksum":0},{"name":"sdl","state":"ONLINE","read":0,"write":0,"checksum":0},{"name":"sdj","state":"ONLINE","read":0,"write":0,"checksum":0},{"name":"mirror-2","state":"ONLINE","read":0,"write":0,"checksum":0},{"name":"sdn","state":"ONLINE","read":0,"write":0,"checksum":0},{"name":"sdm","state":"ONLINE","read":0,"write":0,"checksum":0},{"name":"mirror-3","state":"ONLINE","read":0,"write":0,"checksum":0},{"name":"sdg","state":"ONLINE","read":0,"write":0,"checksum":0},{"name":"sdi","state":"ONLINE","read":0,"write":0,"checksum":0},{"name":"spares","state":null,"read":null,"write":null,"checksum":null},{"name":"sdf","state":"AVAIL","read":null,"write":null,"checksum":null}],"errors":"No known data errors"}]
