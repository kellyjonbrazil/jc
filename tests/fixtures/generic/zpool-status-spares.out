  pool: tank
 state: ONLINE
  scan: scrub repaired 0B in <snip>
config:

	NAME        STATE     READ WRITE CKSUM
	tank        ONLINE       0     0     0
	  mirror-0  ONLINE       0     0     0
	    sdh     ONLINE       0     0     0
	    sdk     ONLINE       0     0     0
	  mirror-1  ONLINE       0     0     0
	    sdl     ONLINE       0     0     0
	    sdj     ONLINE       0     0     0
	  mirror-2  ONLINE       0     0     0
	    sdn     ONLINE       0     0     0
	    sdm     ONLINE       0     0     0
	  mirror-3  ONLINE       0     0     0
	    sdg     ONLINE       0     0     0
	    sdi     ONLINE       0     0     0
	spares
	  sdf       AVAIL

errors: No known data errors
